build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -171,7 +171,9 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
         {
             var hostVersion = c.BuildContext.Get<HostVersion>("HostVersion");
             var configuration = c.BuildContext.Get<string>("Configuration");
-            string rid = c.BuildContext.Get<string>("TargetRID");
+            string rid = DotNetCli.Stage0.GetRuntimeId();
```

Comment:
@ianhays When I used below codes, cross build worked normally. I think the logic to read TARGETRID is correct only when cross build.
```
            if (string.Equals(crossEnv, "1"))                                   
            {                                                                   
                rid = c.BuildContext.Get<string>("TargetRID");                  
            }
```